blog/2021-09-07-introducing-capact.md

```diff
@@ -0,0 +1,124 @@
+cluster - maybe by saying "I want any Kubernetes cluster" and letting magic happen, you could cover both local development (e.g. using [K3d](https://k3d.io)) and production (e.g. Google Cloud GKE) scenarios...
+
+What if I told you... **it is all possible**?
+
+<!--truncate-->
+
+Introducing Capact, crowdsourced expertise. Learn once, use everywhere.
+
+## What is Capact?
+
+Capact is a new way to manage applications and infrastructure and run arbitrary workflows. 
+
+![Here we go again GIF](./assets/here-we-go-again-again.gif) <center><small>source: <a href="https://tenor.com/view/here-wego-again-again-ohboy-sigh-captain-america-gif-15496881" target="_blank">tenor.com</a></small></center>
+
+Yes, we get it, you already know a plenty of different tools for that. But Capact is different. How different?
+
+Capact comes with truly unique feature - **dependencies interchangeability**. If you know the concept of interfaces from a variety of programming languages, you get the idea. Think about Capact as a way of executing workflows with steps, which are described with interfaces. An implementation for a given interface is resolved dynamically in runtime, according to your preferences. Also, these dynamic workflow steps are building blocks, which can be reused by others.
+
+Initially, we are focused on day-one and day-two operations for managing applications and infrastructure. However, Capact is built with universal concepts in mind, which means you can run any arbitrary workflow you want. Data processing, system configuration, cluster benchmarking, auditing, running serverless workloads - the possibilities are virtually endless.
+
+The best thing is that Capact is **fully open source** under the Apache 2.0 license. How cool is that? 😎
+
+## Capact concept in one minute
+
+The whole concept sounds confusing? Let's see that in action.
+
+Let's take [Mattermost](https://mattermost.com/) installation as an example. Mattermost is an open-source alternative for Slack.
+
+Mattermost is a Go server application, which uses PostgreSQL database. If we want to install Mattermost, this how the workflow would look like:
+
+![Mattermost installation 1](./assets/capact-example1.svg)
+
+We need a PostgreSQL database first, to run Mattermost server. It's pretty simple. 
+
+Now, what if we make the PostgreSQL database installation abstract? Think about that as an empty slot, where you can put a puzzle with a specific shape.
+
+![Mattermost installation 2](./assets/capact-example2.svg)
+
+Assuming there are many puzzles out there, you see that only a few of them fit in the slot:
+
+![Mattermost installation 3](./assets/capact-example3.svg)
+
+In the case of PostgreSQL installation, there are four puzzles that fit:
+
+![Mattermost installation 4](./assets/capact-example4.svg)
+
+1. Install PostgreSQL Helm chart on a Kubernetes cluster
+1. Provision GCP CloudSQL for PostgreSQL database
+1. Provision AWS RDS for PostgreSQL database
+1. Reuse existing PostgreSQL database
+
```

Comment:
Maybe it will be good to mention in one sentence how those blocks are/can be selected because it's hard to imagine how this may work.

---

[Policies](https://capact.io/docs/feature/policies/overview) control, which dependency block is chosen. You can have different Policy for your CI, stage and production clusters to select an appropriate Implementation and our Engine will do the rest.